config/locales/provider_interface/reasons_for_rejection.yml

```
@@ -144,6 +147,11 @@ en:
             safeguarding_concerns_other_details:
               blank: Enter details about safeguarding
               too_long: Details about safeguarding must be 100 words or fewer
+            cannot_sponsor_visa_y_n:
+              blank: Select yes if it was because you cannot sponsor the candidate‘s visa
```

🔍  `‘` vs `’` – we should use `‘` to open quotes and `’` to close them (and also as apostrophes).
```suggestion
              blank: Select yes if it was because you cannot sponsor the candidate’s visa
```
(and in the lines below too)

---

Good spot! It turns out i was doing `option + ]` instead of `option + shift + ]`